src/foundry/client/pixi/layers/effects/weather/particles/effect.d.ts

```diff
@@ -0,0 +1,48 @@
+export {};
+
+declare global {
+  /**
+   * An interface for defining particle-based weather effects
+   */
+  class ParticleEffect extends FullCanvasObjectMixin(PIXI.Container) {
+    /**
+     * @param options - Options passed to the getParticleEmitters method which can be used to customize
+     *                  values of the emitter configuration. (default: `{}`)
+     */
```

Comment:
This extra newline might make the TSDoc not "attach" to the constructor. Either way you should remove it.